libmscore/scorefile.cpp

```diff
@@ -433,10 +436,11 @@ bool MasterScore::saveFile(bool generateBackup)
             // remove old backup file if exists
             // remove the backup file in the same dir as score (the traditional place) if exists
             //
-            QString backupDirString = info.path() + QString(QDir::separator()) + ".backup";
+            QString backupPath = preferences.getString(PREF_APP_BACKUP_PATH);
```

Comment:
For consistency, `backupPath` should be `backupSubdirString`.

Should also be `const QString`.